src/main/java/org/zalando/pazuzu/feature/FeatureDto.java

```diff
@@ -48,4 +58,19 @@ public String getTestInstruction() {
     public void setTestInstruction(String testInstruction) {
         this.testInstruction = testInstruction;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Also if it's cumbersome in some way, please stick to code formatting style used in whole codebase. So, no one line if blocks. Always surround with { ... } and put block of if statement on new line.